src/party_menu.c

```diff
@@ -1024,12 +1024,66 @@ static const u8 *const sSelectionStringTable[] =
 };
 
 // Unknown unused data. Feel free to remove.
-static const u16 gUnknown_08615B94[] =
```

Comment:
What does this have to do with modern gcc?

Reply:
whoops

also, i have no idea why it's FAILED; it's OK on my end :thonk: